tests/unit/symbol/test_symbol_subgraph.py

```diff
@@ -0,0 +1,60 @@
+import os
+import pickle
+from unittest import mock
+
+import networkx as nx
+import pytest
+from google.protobuf.message import Message
+
+from automata.cli.cli_utils import initialize_py_module_loader
+from automata.singletons.dependency_factory import DependencyFactory
+from automata.symbol import SymbolGraph
+
+
+class MockProtoBuf(Message):
+    def __init__(self, *args, **kwargs):
+        self.documents = []
+
+
+class MockDocument:
+    def __init__(self, *args, **kwargs):
+        self.symbols = []
+        self.occurrences = []
+
+
+@pytest.fixture
+def symbol_graph():
```

Comment:
looks good, maybe just call this `symbol_graph_mocked_index` or something, just because there is a global fixture `symbol_graph` in the conftest which gets imported.